build.sbt

```diff
@@ -2,22 +2,23 @@ name := "Akka SMPP"
 
 organization := "akkasmpp"
 
-version := "0.3.0-SNAPSHOT"
+version := "0.4.0-SNAPSHOT"
 
-scalaVersion := "2.11.8"
+scalaVersion := "2.12.1"
 
-resolvers += "Typesafe Repository" at "http://repo.typesafe.com/typesafe/releases/"
+crossScalaVersions := Seq("2.11.9", "2.12.1")
 
+resolvers += "Typesafe Repository" at "http://repo.typesafe.com/typesafe/releases/"
```

Comment:
I'm not sure, will try without it.

Regarding publishing - the current namespace (akkasmpp) seems OK. The process is a pain in the ass, though. You'll have to register on sonatype and read like 10 random articles on the internet to get it right, so be ready to spend quite some time on it. I did it recently, so feel free to contact me if you'll feel the need.